pull_twitter.py

```diff
@@ -46,5 +65,28 @@
     # tweepy client
     client = Client(bearer_token=os.environ['TW_BEARER_TOKEN'], wait_on_rate_limit=True)
 
+    # Create output directories
+    dt_fmt = '%Y-%m-%d %H:%M:%S' if os.name == 'posix' else '%Y-%m-%d %H.%M.%S'
+    timestamp = datetime.now().strftime(dt_fmt)
+    subcommand_dir = f"{str(config.local.output_dir)}/{args['name']}"
+    output_time_dir = f"{subcommand_dir}/{timestamp}"
+    if not os.path.isdir(subcommand_dir):
+            os.mkdir(subcommand_dir)
+
+    data_dir, metadata_dir = f"{output_time_dir}/data", f"{output_time_dir}/metadata"
+    os.makedirs(output_time_dir)
+    os.mkdir(data_dir)
+    os.mkdir(metadata_dir)
+
+    # Save query metadata
+    copyfile(args['config_file'], f"{metadata_dir}/config.yaml")
+    with open(f"{metadata_dir}/command.txt", "w") as command_file:
+        command_file.write(" ".join(sys.argv))
+
+    # Clean command keyword arguments
+    ignore_args = ['config_file', 'name', 'func']
+    command_kwargs = {key:value for key, value in args.items() if (not key in ignore_args) and (value)}
+    command_kwargs['output_dir'] = data_dir
+
     # Run the application with parsed configuration and initialized client
-    args['func'](config, client)
\ No newline at end of file
+    args['func'](config, client, **command_kwargs)
```

Comment:
Two minor points:
1. Could be nice to print a new-line here so that the terminal prompt starts on a new line rather than on the end of the last output
2. Best practice to add a blank line at end of file